iron/index.html

```diff
@@ -48,7 +48,7 @@ <h1 class="jumbotron-heading display-1">Iron</h1>
 allocated threads. In particular, Iron can be used to show the correctness of challenging examples, where the
 reclamation of memory is delegated to a forked-off thread. We show soundness of Iron by means of a model of
 Iron, defined on top of the Iris base logic, and we use this model to prove that memory resources are accounted
-for precisely and not leaked. We have formalized all of the developments in the Coq proof assistant.
+for precisely and not leaked. We have formalized all of the developments in the Rocq proof assistant.
```

Comment:
This is also a historic page, isn't it? You "own" this page so I won't block changing it, but unless you plan to also upload a revised version of the paper I find this an odd move.

If you do change it, please use "Rocq Prover", not "proof assistant".

Reply:
I "own" the project indeed (as the only one of the co-authors who maintains it), so I decided to go for Rocq Prover as you suggested.